### PR TITLE
feat(FEC-11785): [Youbora] - update youbora options for drm system that is used by the player

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

update youbora options for drm system that is used by the player

https://documentation.npaw.com/npaw-integration/docs/setting-options-and-metadata

related prs:
https://github.com/kaltura/kaltura-player-js/pull/515
https://github.com/kaltura/playkit-js/pull/624
https://github.com/kaltura/playkit-js-dash/pull/175

solves: FEC-11785

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/adapter/adapter.js [hunk 1]
```diff
@@ -158,6 +158,9 @@ let YouboraAdapter = youbora.Adapter.extend({
 
   /** @returns {void} - Listener for 'playing' event. */
   playingListener: function () {
+    if (this.player.drmInfo().scheme) {
```
Comment (yairans) [inline src/adapter/adapter.js:161]: `this.player.drmInfo()` could be null

File: src/adapter/adapter.js [hunk 2]
```diff
@@ -158,6 +158,9 @@ let YouboraAdapter = youbora.Adapter.extend({
...
+      this.plugin.options['content.drm'] = this.player.drmInfo().scheme;
```
Comment (yairans) [inline src/adapter/adapter.js:162]: need to think if we want to be aligned with android values. see https://github.com/kaltura/playkit-android-youbora/pull/81

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/adapter/adapter.js (hunk 2 above)
Comment (JonathanTGold) [reply src/adapter/adapter.js]: aligned with android
https://github.com/kaltura/playkit-android/blob/dev/playkit/src/main/java/com/kaltura/playkit/PKDrmParams.java

File: src/adapter/adapter.js [hunk 3]
```diff
@@ -1,7 +1,13 @@
 // @flow
 import youbora from 'youboralib';
 import {core} from 'kaltura-player-js';
-const {Error, MediaType} = core;
+const {Error, MediaType, DrmScheme} = core;
+
+const DrmSchemeName = {
+  FAIRPLAY: 'FairPlay',
+  PLAYREADY: 'PlayReadyClassic',
+  WIDEVINE: 'WidevineClassic'
```
Comment (yairans) [inline src/adapter/adapter.js:9]: remove classic from widevine and playready

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/adapter/adapter.js (hunk 3 above)
Comment (JonathanTGold) [reply src/adapter/adapter.js:9]: removed (note it is different in android https://github.com/kaltura/playkit-android/blob/dev/playkit/src/main/java/com/kaltura/playkit/PKDrmParams.java)

**Review by yairans — DISMISSED** [review]

File: package.json [hunk 4]
```diff
@@ -72,7 +72,8 @@
     "flow-bin": "^0.129.0",
     "husky": "^4.2.5",
     "istanbul": "^0.4.5",
-    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#v1.0.0",
+    "kaltura-player-js": "kaltura/kaltura-player-js#master",
+    "@playkit-js/playkit-js": "kaltura/playkit-js#master",
```
Comment (yairans) [inline package.json:76]:
```suggestion
    "@playkit-js/playkit-js": "canary",
```

File: package.json [hunk 5]
```diff
@@ -99,8 +100,8 @@
     "youboralib": "6.7.42"
   },
   "peerDependencies": {
-    "@playkit-js/playkit-js": "0.63.0",
-    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#v1.0.0"
+    "kaltura-player-js": "kaltura/kaltura-player-js#master",
+    "@playkit-js/playkit-js": "kaltura/playkit-js#master"
```
Comment (yairans) [inline package.json:104]: same

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/adapter/adapter.js [hunk 6]
```diff
@@ -1,7 +1,14 @@
 // @flow
 import youbora from 'youboralib';
 import {core} from 'kaltura-player-js';
-const {Error, MediaType} = core;
+const {Error, MediaType, DrmScheme} = core;
+
+const DrmSchemeTitle = {
+  FAIRPLAY: 'FairPlay',
+  PLAYREADY: 'PlayReady',
+  WIDEVINE: 'Widevine',
```
Comment (yairans) [inline src/adapter/adapter.js:9]:
```suggestion
  PLAYREADY: 'PlayReadyCENC',
  WIDEVINE: 'WidevineCENC',
```
per https://github.com/kaltura/playkit-android/blob/e8b421cce3222d7eb18ef5fb01e24cbc64f2a642/playkit/src/main/java/com/kaltura/playkit/PKDrmParams.java#L25

**Review by yairans — CHANGES_REQUESTED** [review]

**Review by yairans — APPROVED** [review]

**Review by yairans — APPROVED** [review]

**Pull request merged**